# My Jupyter Notebook on IBM Watson Studio

**Christine Shrader**

Data Scientist-ish (this is both what I am and what I want to be, though I'd like to eventually drop the '-ish')

_If you can't have fun with it, then it's not worth doing. Data science is not only useful, but fun (and the money isn't bad either). My passion is anthropology, people and culture are at the heart of everything we do. I want to bring more data science into anthropology and more anthropology into data science._

### The next cell shows an example calculation of the difference in distance from my hometown (Houston, Tx) to the planet Jupiter versus the distance to Jupiter, California.

In [2]:
365000000-1901

364998099

### A summary of my distance difference findings

---

US towns used:
* Jupiter, California
* Jupiter, Florida
* Jupiter, North Carolina

Distance difference calculations (see example above):

|State of Jupiter| Miles from Houston| Miles Difference (vs planet Jupiter)|
|----------------|:-----------------------------:|:---------------------------------------------:|
|California|1901|364998099|
|Florida|1107|364998893|
|North Carolina|1004|364998996|